portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -119,15 +119,7 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 			</div>
 
 			<aui:script senna="temporary" type="text/javascript">
-				(function () {
-					var dropdown = document.getElementById("<%= ariaPagination %>");
-
-					var button = dropdown.querySelector('.dropdown-toggle');
-					var list = dropdown.querySelector('.dropdown-menu');
-
-					var options = list.querySelectorAll('.dropdown-item');
-					var selectedItemValue = button.dataset.attribute;
-
+				function handleDropdownKeyPress(button, list, options, dropdown) {
```

Comment:
It won't hurt if we namespace this function, in case we have several page iterators on the same page, we'll have separate functions. In theory, it shall not matter as the `handleDropdownKeyPress()` function is not dependant of any external information, however it remains as a common pattern